tripy/tests/frontend/test_utils.py

```diff
@@ -59,7 +59,7 @@ def func(a: tp.types.TensorLike):
         a = func(1.0)
 
         assert isinstance(a, tp.Tensor)
-        assert a.stack_info[2].column_range == (17, 20)
+        assert a.stack_info[4].column_range == (17, 20)
```

Comment:
It seems that the the slice method is registered twice, once due to being in the tensor method registry and a second time probably because of the `export.public_api` decorator's being applied to all methods of `Tensor`. Probably not a problem, but amusing.

Reply:
Is there a way to avoid that? It will slow down the code unnecessarily since we'll end up doing type checking twice. We could either have the function registry skip methods that are already part of a function registry, or we could change the `TENSOR_METHOD_REGISTRY` to be just a normal `dict` instead of a `FunctionRegistry`. 

The latter seems cleaner from a design perspective although we will probably then lose the ability to do function overloading for methods (which might be fine since it seems unlikely that we'd want to).

Reply:
The second approach would probably be reasonable since the tensor methods we would register would also be in the public function registry. We would also still be able have overloading by registering them under the same key in the public function registry.

I'll see if there would be a simple way to avoid registering in two registries (this would build in an assumption that we would want functions to be members of only one registry at a time).

Reply:
I went with the approach of turning the tensor method registry into an ordinary dict because I didn't want to introduce convoluted logic for trying to prioritize which registry should be the one that checks overloads. This causes few problems.

Incidentally, the reason for changing the stack depth here was not because of the second registry; it was because `__init__` previously was not registered but now is. However, I checked with other registered methods and confirmed that there were two registry checks without the changes in `[ceefce7](https://github.com/NVIDIA/TensorRT-Incubator/pull/388/commits/ceefce7409bc7efc2ed03e517086976a5e2cc4b4)`.